firefox-ios/firefox-ios-tests/Tests/XCUITests/BaseTestCase.swift

```diff
@@ -375,6 +375,12 @@ class BaseTestCase: XCTestCase {
         startCoordinate.press(forDuration: 0, thenDragTo: endCoordinate)
         endCoordinate.press(forDuration: 0, thenDragTo: startCoordinate)
     }
+
+    func dismissSurveyPrompt() {
```

Comment:
This kind of actions are very flaky because we check if the element exists but because there is not a wait for it.. it could lead to failures because it is not present although it will be... wondering if we could have the surveys off for testing or to have them as a launch argument condition..

Reply:
i agree with you, not sure how to add the launch argument for now, but if could be addressed in a future PR that would be great